Kernel/Graphics/GraphicsDevice.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2021, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+#include <AK/Types.h>
+#include <Kernel/Devices/BlockDevice.h>
+#include <Kernel/Graphics/FramebufferDevice.h>
+#include <Kernel/PhysicalAddress.h>
+
+namespace Kernel {
+class FramebufferDevice;
+class GraphicsDevice : public RefCounted<GraphicsDevice> {
+public:
+    enum class Type {
+        VGACompatible,
+        Bochs,
+        SVGA,
+        Raw
+    };
```

Comment:
Is this required?

Reply:
Technically yes, because `GraphicsManagement` holds `m_vga_adapter` and it determines the appropriate device by looking at the type of it. Later on, we could try to make VGA compatible adapters to work (or to not work) in VGA mode manually.